### ISSUE TITLE
"Mirror Under Door" & possibly other look at object commands stay on screen for too long sometimes

### ISSUE THREAD

**Issue by theengineertcr** [eezstreet/SWATEliteForce#234]
I've had this issue where after I check under a door using an optiwand and then left clicking after changing to another piece of equipment will give me the optiwand again. I will provide video evidence soon.

**Comment by eezstreet** [comment]
This doesn't sound like a bug; it's supposed to equip a piece of equipment when you click while aiming at certain points.

**Comment by sandman332** [comment]
I agree Engineer.  There should be a delay or something.  I hate unequiping my optiwand then it coming back again.

**Comment by theengineertcr** [comment]
Rexplained:You have a box on screen that says left click to equip optiwand. You switch to another equipment. Walk away. As soon as equipment is up, press left click. You'll switch back to optiwand and the text box is still there. My suggestion is to make it show up ONLY if you're looking at it and if you aren't looking at that location anymore while switching equipment, remove the message onscreen.

**Comment by eezstreet** [comment]
Reopened, with a less-confusing name.
